.eslintignore

```diff
@@ -1,6 +1,5 @@
 node_modules
 
-/eslint-plugin-local
```

Comment:
Why you removed it? It was needed for bem-cn-lite?

Reply:
It's local plugin that didn't connect to project and it may checking that `bem-cn-lite` didn't import directly. I think, It's not need any more